### PR TITLE
LayerManager: additional layer functions

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
additional functions:

- duplicate Layer
- expand all Layer
- collapse all Layer

### CODE REVIEW

File: src/modules/layerManager/i18n/layerManager.provider.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ export const provide = (lang) => {
 				layerManager_expand: 'expand properties',
 				layerManager_move_up: 'move Layer up',
 				layerManager_move_down: 'move Layer down',
-				layerManager_remove: 'remove Layer'
```
Comment (taulinger) [inline src/modules/layerManager/i18n/layerManager.provider.js:14]: Would suggest that we name this process consistently "to clone"

File: src/modules/layerManager/i18n/layerManager.provider.js [hunk 2]
```diff
@@ -11,7 +11,12 @@ export const provide = (lang) => {
...
+				layerManager_remove: 'remove Layer',
+				layerManager_duplicate: 'duplicate Layer',
```
Comment (taulinger) [inline src/modules/layerManager/i18n/layerManager.provider.js:15]: clone layer

File: src/modules/layerManager/i18n/layerManager.provider.js [hunk 3]
```diff
@@ -11,7 +11,12 @@ export const provide = (lang) => {
...
+				layerManager_layer_copy: 'Copy',
```
Comment (taulinger) [inline src/modules/layerManager/i18n/layerManager.provider.js:16]: clone

**Review by taulinger — COMMENTED** [review]
Very nice addition!

File: src/modules/layerManager/i18n/layerManager.provider.js (hunk 3 above)
Comment (thiloSchlemmer) [reply src/modules/layerManager/i18n/layerManager.provider.js:16]: the key `layerManager_layer_copy` is used as suffix for the label of a cloned layer, i guess '[Layername] Copy' is more familiar to the user than '[Layername] Clone'

**Pull request merged**